### PR TITLE
Update roadmap for 1.7

### PR DESCRIPTION
**Pull request by justinsb**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2625)

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -28,4 +8,34 @@
  
 ## Other features we are working on in the 1.7 timeframe
 
+* etcd controller to allow moving between versions
 * kops server for better team scenarios
+* support for bare-metal
```
Comment (chrislovecnm) [inline ROADMAP.md]: Isn't this another provider?

File: ROADMAP.md [hunk 2]
```diff
@@ -28,4 +8,34 @@
  
 ## Other features we are working on in the 1.7 timeframe
 
+* etcd controller to allow moving between versions
```
Comment (chrislovecnm) [inline ROADMAP.md]: Are we doing the controller?

File: ROADMAP.md [hunk 3]
```diff
@@ -28,4 +8,34 @@
...
 * kops server for better team scenarios
+* support for bare-metal
+* more gossip backends
```
Comment (chrislovecnm) [inline ROADMAP.md]: For what discovery?

File: ROADMAP.md [hunk 4]
```diff
@@ -28,4 +8,34 @@
...
+* IAM integration
+* more cloud providers
+* promote GCE to stable
```
Comment (chrislovecnm) [inline ROADMAP.md]: Some ideas:

- instance groups support (delete)
- rolling-update

File: ROADMAP.md (hunk 2 above)
Comment (justinsb) [inline ROADMAP.md]: Unless the etcd-controller meets requirements.

File: ROADMAP.md (hunk 1 above)
Comment (justinsb) [inline ROADMAP.md]: A good point, but I don't think most people think of it that way ;-)

File: ROADMAP.md (hunk 3 above)
Comment (justinsb) [inline ROADMAP.md]: S3 is now the only piece left.  If we could replace that with something gossip, that would be useful.

File: ROADMAP.md (hunk 4 above)
Comment (justinsb) [inline ROADMAP.md]: Bringing rolling-update out of a alpha is a great suggestion.  WIll add.

Not sure what instance groups delete is?  We have instance-group deletion.

Comment (chrislovecnm) [inline ROADMAP.md]: I was actually mentioning missing pieces in gce. Instance groups commands do not work ;)

Yes improvements to rolling-update are are on the way! Already have one pr.

**Comment by chrislovecnm** [comment]
Can we get a squash and are we good to merge?

**Pull request merged**